tests/components/AppliedFilters.test.tsx

```diff
@@ -259,7 +260,7 @@ describe('AppliedFilters with hierarchical facets', () => {
     expect(filterPills).toHaveLength(3);
 
     const fruitButton = screen.queryByLabelText('Remove "food" filter');
-    userEvent.click(fruitButton);
+    fruitButton && userEvent.click(fruitButton);
```

Comment:
for these cases where we are certain the element must be defined, I believe we can just use `getBy...` functions instead of `queryBy...` so we don't need to check for undefined (e.g. use `getByLabelText` here so we don't need the `fruitButton &&` part)